packages/web-react/src/components/Collapse/demo/CollapseHideTrigger.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Collapse from '../Collapse';
+import { useCollapse } from '../useCollapse';
+
+const CollapseHideTrigger = () => {
+  const { isOpen, toggleHandler } = useCollapse(false);
+
+  return (
+    <p>
+      Quis orci molestie, condimentum metus a lorem lorem fringilla sem lobortis libero rhoncus nullam, purus sapien
+      aenean sed id tortor. Molestie quam, lectus quis ac nec suspendisse consectetur finibus a accumsan, bibendum dolor
+      sit amet congue rhoncus erat. Semper turpis, varius iaculis venenatis ut congue sollicitudin nulla, convallis nibh
+      maximus erat id aliquam. Nec interdum, auctor elit vitae pulvinar fusce gravida curabitur odio leo, nunc a mauris
+      auctor. Vel integer, placerat pulvinar nisi nunc donec felis erat quis, phasellus ac nec phasellus ac nibh.
+      Scelerisque fringilla, viverra nisl vel purus sed porttitor porttitor et iaculis commodo, vehicula urna purus
+      ultrices.{' '}
+      <a href="#" role="button" onClick={toggleHandler} aria-expanded={isOpen}>
+        <span className="accessibility-closed">…more</span>
```

Comment:
VisuallyHidden?

Reply:
our component VisuallyHidden does not handle `accessibility-closed` class

Reply:
Should this be prefixed?

Reply:
I think this is ok since this is only the demo.